recipe/build.sh

```diff
@@ -1,20 +1,28 @@
 #!/bin/sh
 
-mkdir build && cd build
+if [[ ${HOST} =~ .*darwin.* ]]; then
+  # Adapt cflags for the sdk in use
+  export CFLAGS="${CFLAGS} -isysroot ${SDKROOT:-$CONDA_BUILD_SYSROOT}"
 
-# DYLIB_COMPAT_VERSION is set for compatibility with autotools builds
-# See https://github.com/conda-forge/sdl-feedstock/pull/6#issuecomment-1065039606
-cmake ${CMAKE_ARGS} -GNinja -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  # Additional build option depending on target architecture
+  if [[ "${target_platform}" == "osx-arm64" ]]; then
+    export ADDITIONAL_OPTIONS="-DCMAKE_OSX_ARCHITECTURES=arm64 -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0"
```

Comment:
`CMAKE_OSX_DEPLOYMENT_TARGET` is already defined in `CMAKE_ARGS` env variable (see the build log), what is the need to redefine it?

Reply:
Is it also true for `CMAKE_OSX_ARCHITECTURE` ? `CMAKE_OSX_SYSROOT=path_to_sdk` ?
Where can I find the list of variables set by `CMAKE_ARGS` ?

Reply:
Nevermind I found it indeed in build logs, thank you !

Reply:
I suspect also `CMAKE_OSX_ARCHITECTURE` is not necessary, unless sdl3 has something specific in its CMake code that differentiate it from other CMake projects.

Reply:
Just removed all options present in `CMAKE_ARGS` and add a comment about `CMAKE_OSX_ARCHITECTURE `